src/main/java/seedu/address/model/Goods.java

```diff
@@ -0,0 +1,29 @@
+package seedu.address.model;
+
+/**
+ * Deals with the name of the product to be delivered.
+ */
+
+public class Goods {
```

Comment:
Maybe can consider adding more stuff to the `Goods` class. For example, a driver needs to deliver multiple items to one person, so can consider maintaining a list of items in the `Goods` class, and each item has a quantity, maybe. 

In the end, you just need to format the `toString()` method nicely so we can just use the string representation of it.

Reply:
Right thanx for the suggestion